examples/complete/main.tf

```diff
@@ -20,7 +20,18 @@ data "aws_eks_cluster_auth" "this" {
   name = module.eks_blueprints.eks_cluster_id
 }
 
-data "aws_ecr_authorization_token" "token" {}
+# Equivalent of aws ecr get-login
+data "aws_ecr_authorization_token" "ecr_token" {}
+# Multiple docker push commands can be run against a single token
+resource "null_resource" "renew_ecr_token" {
```

Comment:
instead of this which requires the awscli and docker CLI to be installed local to where terraform is running, we should update each ACK addon to pass the creds to the `helm_release` resource via the `helm_config` like this https://github.com/aws/karpenter/pull/2913/files#diff-c996548a210a321770151def6b6f02b6204a8bcdda9a1e1d86973fa3fbda5e18R263-R264

Reply:
moved to helm release.